cf-agent/simulate_mode.c

```diff
@@ -63,8 +63,35 @@ static inline void PrintDelimiter()
 static void ManifestStatInfo(const struct stat *st)
 {
     assert(st != NULL);
+
+    errno = 0;
     struct passwd *owner = getpwuid(st->st_uid);
+    if (owner == NULL)
+    {
+        if (errno == 0)
+        {
+            Log(LOG_LEVEL_ERR, "Failed to get file owner name: non-existent user id %d", st->st_uid);
+        }
+        else 
+        {
+            Log(LOG_LEVEL_ERR, "Failed to get file owner name: %s", GetErrorStr());
+        }
+        return;
```

Comment:
I'd prefer if only the UID/GID-related bits were  omitted in the output. We can still report everything else from the stat info.

Reply:
I don't see how was my comment resolved in the code changes. :wink:

Reply:
Sry @vpodzime , I must have misunderstood. I removed UID/GID from error output, did you mean to omit it in stdout output? Could you please elaborate?

Reply:
Maybe confusion between omit / emit ? Opposites ;)